### PR TITLE
[BOP-979] Enhance message kit to accept timestamp object for hover

### PR DESCRIPTION
**Pull request by haydenrou**
https://app.abstract.com/share/00ce8869-748b-4ac2-9d38-07083137a6a6?collectionId=afd8ed52-e16b-4afd-b503-f8bc6cf2e80b&collectionLayerId=76a6e707-0640-4352-a903-b37714f4be41&present=true&preview=false&sha=32ff4fbf2dce3bedb52ab8f7e2261e3479cdb658

#### Screens

Normal:
Screenshot 2021-06-08 at 7 26 20 AM

On hover:
Screenshot 2021-06-08 at 7 27 05 AM

#### Breaking Changes

No breaking changes. A change was made that you can now pass this kit a `timestamp_object`, which is like `DateTime.current`. When this is present, the hover effect will work, otherwise it will not.

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/BOP-979

#### Checklist:

- [ ] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [ ] **URGENCY** Please select a release milestone
- [ ] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [ ] **SCREENSHOT** Please add a screen shot or two.
- [ ] **SPECS** Please cover your changes with specs.
- [ ] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*

### CODE REVIEW

File: playbook/app/pb_kits/playbook/pb_message/_message.jsx [hunk 1]
```diff
@@ -56,18 +58,33 @@ const Message = (props: MessageProps) => {
         <Avatar
             imageUrl={avatarUrl}
             name={avatarName}
-            size="sm"
+            size="xs"
```
Comment (thestephenmarshall) [inline playbook/app/pb_kits/playbook/pb_message/_message.jsx:61]: Was this an intended change?

**Review by thestephenmarshall — COMMENTED** [review]
It looks like some of the structure has changed in this kit (which is fine), but when I view the deployed env vs. prod things don't exactly line up with the screenshots

Prod:
Screen Shot 2021-06-08 at 12 50 47 PM

Review Env:
Screen Shot 2021-06-08 at 12 50 57 PM

I much prefer these changes over prod, but I will leave the final decision to other reviewers.

**Comment by haydenrou** [comment]
@thestephenmarshall I'm not sure I understand what you mean. I see the changes in the milano environment to reflect the screenshots I provided: 
Screenshot 2021-06-08 at 11 19 11 AM

File: playbook/app/pb_kits/playbook/pb_message/_message.jsx (hunk 1 above)
Comment (haydenrou) [reply playbook/app/pb_kits/playbook/pb_message/_message.jsx:61]: Yes, by @evalouderback

**Comment by evalouderback** [comment]
The PR is for the timestamp on hover, but also includes an updated comment design for Runway and Connect @thestephenmarshall

**Comment by thestephenmarshall** [comment]
@haydenrou weird hmm... disregard?

**Comment by megantrimble** [comment]
> The PR is for the timestamp on hover, but also includes an updated comment design for Runway and Connect @thestephenmarshall

@evalouderback Did you look at any treatments for the time stamp next to them name? Or possible options for different alignments? Just thinking about how this message kit will translate to other areas of Nitro.

**Comment by haydenrou** [comment]
New commits change some styles to look like the following, and separate out the new timezone object example to it's own section. See image:
Screenshot 2021-06-08 at 2 18 41 PM

**Comment by jasoncypret** [comment]
This is great @haydenrou. I gave @megantrimble & @evalouderback some specific feedback to relay. We are very close on this!!!!

**Comment by haydenrou** [comment]
Screenshot 2021-06-11 at 11 29 40 AM
I've made an example for Megans usecase in Connect

**Comment by jasoncypret** [comment]
Great job @haydenrou !!! These additions are great and support 2 completely different projects at the same time. 🙏

**Review by jasoncypret — APPROVED** [review]

**Review by jasperfurniss — APPROVED** [review]

**Pull request merged**